src/client/index.tsx

```diff
@@ -1,18 +1,29 @@
 import React from "react";
 import ReactDOM from "react-dom";
 
+interface ConsultantParameter {
+    label: string;
+    value: string;
+}
 
-const name = "John Doe";
+const consultantLines: ConsultantParameter[] = [{
+    label: "Name",
+    value: "John Doe",
+}]
```

Comment:
All good here, it was difficult to explain on the phone, but just use this format in the future:
```
const consultantLines: ConsultantParameter[] = [
    {
        label: "Name",
        value: "John Doe",
    },
];
```

Reply:
Okay!